### ISSUE TITLE
Translation

### ISSUE THREAD

**Issue by jgroenen** [flore2003/staythefuckhome#3]
I can translate to Dutch. What would be the way to do this. Please describe in readme.

**Comment by jgroenen** [comment]
@flore2003 can you help?

**Comment by vmoshynskyi** [comment]
Hi @jgroenen.
Briefly:
1. Fork/clone the repo.
2. Create a copy of `site/index.njk` (or `site/de.njk` could be better in your case) and put it under `site/nl.njk`.
3. Add language into `site/globals/langs.json` like the other languages.
4. To test in browser run `npm install` then `npm run dev`.
5. Create  pull request.
6. You can find example [here](https://github.com/flore2003/staythefuckhome/pull/4/files).

Feel free to ask questions.

**Comment by jgroenen** [comment]
I started translating to Dutch.

**Comment by jgroenen** [comment]
I created a translation, but needs review: https://github.com/jgroenen/staythefuckhome

**Comment by flore2003** [comment]
@jgroenen We actually have two more translations for Dutch and are right now at the point that we don't know which to choose. Do you mind looking at them too?

**Comment by jgroenen** [comment]
No problem

**Comment by jgroenen** [comment]
If you would, you can send the translators to doemee.codefor.nl (the Dutch civic tech network)

**Comment by ahunts** [comment]
Hi, this is the version we have gotten from our translators, can you take this as input for your branch and update it?

```
Onze regering doet niet wat ze zou moeten doen, dus nu moeten wij zelf in actie komen! Neem je verantwoordelijkheid en blijf verdomme thuis! 

Our governments are failing us, we need to take action now! Do your part and stay the fuck home!

Een beweging om de COVID-19 pandemie te stoppen
A Movement to Stop the COVID-19 Pandemic

Onze regeringsleiders doen niet genoeg om de verspreiding van SARS-CoV-2 te voorkomen en de COVID-19 pandemie in te dammen. Trage reacties, beleid gericht op geruststelling van het publiek, en de neiging de economie te stabiliseren weerhoudt hen ervan de maatregelen te nemen die nodig zijn om miljoenen mensen tegen dit virus te beschermen. Het is tijd dat wij, als burgers van deze aarde, in actie komen en
onze verantwoordelijkheid nemen in dit gevecht tegen COVID-19.

Bot gezegd: Blijf Verdomme Thuis!

Our governments are failing us in preventing the spread of SARS-CoV-2 and containing the COVID-19 pandemic. Slow reactions, public appeasement policies, and their urge to stabilize the economy are keeping them from taking the measures it takes to protect millions from this disease. It is time for us, as citizens of this earth, to take action now and do our part in fighting COVID-19.
Let's put it bluntly: Stay The Fuck Home!
Het Zelf-Quarantaine Manifesto 
The Self-Quarantine Manifesto
In deze situatie zonder goed bestudeerde behandeling en met een vaccin dat pas over minimaal een jaar beschikbaar komt, is de enige effectieve manier om de coronavirus epidemie een halt toe te roepen het virus minder kans te geven zich te verspreiden. De volgende lijst met acties, in volgorde van makkelijk te implementeren tot meest effectief in het gevecht tegen de pandemie, kan dienen als losse richtlijn voor mensen die onderdeel willen worden van de beweging en in actie willen komen waar onze leiders tekort schieten.
With no well-studied treatment and a viable vaccine still being out at least another year, the only effective way to keep the coronavirus epidemic at bay is to give the virus less chances of spreading. The following list of actions, ordered from easiest to implement to most effective in the fight against the pandemic, should serve as a set of loose guidelines for people who wish to join the movement and take action, where the people in charge continue to neglect.
1	Raak niet in paniek, maar wees wel alert
2	Was vaak je handen en gebruik goede hoest en nies etiquette
3	Probeer je gezicht zo min mogelijk aan te raken, inclusief je mond, neus en ogen
4	Houd sociale afstand, geen geknuffel of gekus, geef geen hand en ook geen high fives. Gebruik eventueel een veiliger alternatief.
5	Ga niet naar concerten, theater, sport evenementen, of enig ander massa evenement.
6	Bezoek geen musea, exposities, bioscopen, nachtclubs of andere locaties met veel bezoekers
7	Blijf weg bij sociale bijeenkomsten zoals in clubs, bij kerkdiensten of feestjes bij vrienden
8	Reis niet onnodig en beperk reis afstanden waar mogelijk
9 	Vermijd het openbaar vervoer
10	Waar mogelijk, werk van thuis. Dring er bij werkgevers op aan dit mogelijk te maken.
11 	Vervang zoveel mogelijk sociaal contact door alternatieven zoals bellen of video-oproep
12	Verlaat je huis zo min mogelijk
 
Don't panic, but be alert.
Wash your hands often and practice good cough and sneeze etiquette.
Try to touch your face as little as possible, including your mouth, nose, and eyes.
Practice social distancing, no hugs and kisses, no handshakes, no high fives. If you must, use safer alternatives.
Do not attend concerts, stage plays, sporting events, or any other mass entertainment events.
Refrain from visiting museums, exhibitions, movie theaters, night clubs, and other entertainment venues.
Stay away from social gatherings and events, like club meetings, religious services, and private parties.
Reduce your amount of travel to a minimum. Don't travel long distances if not absolutely necessary.
Do not use public transportation if not absolutely necessary.
If you can work from home, work from home. Urge your employer to allow remote work if needed.
Replace as many social interactions with remote alternatives like phone calls or video chat.
Do not leave your home if not absolutely necessary.
Onthoud hierbij dat er geen goede of slechte hoeveelheid actie is. Doe de dingen die je kunt doen en zonder dat je inkomsten in gevaar worden gebracht. Neem hiervoor geen ontslag! Onthoud ook dat alle bovenstaande acties bijdragen.
Please keep in mind there is no right or wrong amount of actions to take. Only take the actions that you feel comfortable with and that do not endanger your livelihood. Do not quit your job over this! However, keep in mind that every action helps.
Waarom dit zo belangrijk is
SARS-CoV-2 is een zeer besmettelijk en potentieel dodelijk virus dat de luchtwegaandoening genoemd COVID-19 veroorzaakt. Je kent het misschien onder een van de vele andere namen, waaronder 2019-nCov, nieuw coronavirus, Wuhan coronavirus, China of Wuhan griep, of simpelweg coronavirus. Al deze namen refereren aan hetzelfde virus dat deze beweging probeert te stoppen.
Why it's so Important
SARS-CoV-2 is a highly infectious and potentially deadly virus that causes a respiratory disease called COVID-19. You might know it under one of its many other names, including 2019-nCoV, novel coronavirus, Wuhan coronavirus, China or Wuhan flu, or just simply coronavirus. All of these refer to the same virus that this movement is trying to stop.
In de laatste maanden en weken, werd het virus en de ziekte die het veroorzaakt steeds vergeleken met influenza en de jaarlijkse griep. Door overeenkomsten in symptomen en schijnbaar vergelijkbare ernst, is dit voor veel mensen een logische vergelijking. Echter, als we kijken naar wat we nu over dit nieuwe virus weten, kunnen we het gevaar voor onze maatschappij moeilijk ontkennen.
Over the past months and weeks, the virus and corresponding disease have been compared over and over to the influenza virus and the seasonal flu. Due to the overlap in symptoms and seemingly similar severity, this comparison comes naturally to a lot of people, however, looking at what we know so far about this new virus, the threat it poses to society is not easily brushed aside.
1	Besmettelijker dan griep
Met een geschat R0 tussen 1.4(USE THE EM DASHES)6.49 en een geschat gemiddelde van 3.28 [1], is SARS-CoV-2 veel besmettelijker en verspreidt zich veel sneller dan de jaarlijkse griep, wat een mediaan R0 heeft van 1.28[2].
2	Dodelijker dan griep
Het zogenaamde case fatality rate (CFR) van SARS-CoV-2 ligt waarschijnlijk rond de 2%[3], wat betekent dat naar schatting 2% van de mensen die de diagnose COVID-19 krijgen hier helaas aan zullen overlijden. Ter vergelijking wordt de CFR van de jaarlijkse griep geschat op ongeveer 0.1%[4], wat zou betekenen dat SARS-CoV-2 naar schatting 20 keer zo dodelijk is als de griep.
3	Mogelijk ernstige ziekteverschijnselen
Naar schatting zullen 15(USE THE EM DASHES)20% van de geïnfecteerde mensen ernstige ziekteverschijnselen vertonen die medische hulp vereisen[5], zoals longontsteking, kortademigheid en verlaagde zuurstofsaturatie in het bloed.
4	Geen behandeling, geen vaccin, geen weerstand
Omdat SARS-CoV-2 net nieuw is, is er geen goed onderzochte behandeling voor COVID-19 en is er meer onderzoek nodig voordat een effectieve behandeling wordt gevonden. Ook is er nog geen vaccin voor SARS-CoV-2 [6] en zal de ontwikkeling van zo’n vaccin nog enige tijd duren. Zonder vaccin en zonder wijd verspreide weerstand tegen dit nieuwe virus is iedereen vatbaar voor besmetting. Terwijl de meeste geïnfecteerde mensen milde symptomen zullen ervaren, kan dit gebrek aan groepsimmuniteit ertoe leiden dat een grote groep kwetsbare personen ernstige symptomen zullen krijgen.
5	Exponentiële groei
Doordat er geen immuniteit of weerstand is tegen dit virus, is de gehele bevolking de bovenste limiet voor het aantal mogelijke besmettingen met dit virus. En hoewel exponentiële groei altijd langzaam lijkt te gaan in het begin, kan het tot ondenkbaar grote aantallen leiden in een zeer korte tijd. Nu een verdubbeling van het aantal geïnfecteerden een kwestie van een paar dagen[7] is, kan het medisch systeem overbelast raken, waardoor er meer sterfgevallen zullen zijn omdat mensen niet de zorg krijgen die zij nodig hebben. 
More Contagious Than the Flu
With an estimated R0 between 1.4 – 6.49 and a mean estimate of 3.28[1], SARS-CoV-2 is much more infectious and spreads much faster than the seasonal flu, which has a median R0 of 1.28[2].
More Deadly Than the Flu
The so-called case fatality rate (CFR) of SARS-CoV-2 is estimated to be at around 2%[3], meaning that unfortunately, an estimated 2% of people getting diagnosed with COVID-19 will succumb to it. In comparison, the CFR of seasonal influenza is estimated to be around 0.1%[4], this means SARS-CoV-2 is estimated to be about 20 times more deadly than the seasonal flu.
Possibility of Severe Symptoms
An estimated 15 – 20% of infected individuals suffer from severe symptoms that require medical attention[5], including pneumonia with shortness of breath and lowered blood oxygen saturation.
No Treatment, No Vaccine, No Immunity
As SARS-CoV-2 has only recently emerged, there is no well-studied treatment for COVID-19 and more research is needed, in order to treat infected people efficiently. Likewise, there is no vaccine for SARS-CoV-2 yet[6] and the development of such vaccine will take a significant amount of time. With no vaccines and no prior and wide-spread immunity, anyone is a susceptible target for infection. While most infected people will only suffer from mild symptoms, this lack of herd-immunity can lead to severe illness in a significant amount of at-risk individuals.
Exponential Growth
Due to the lack of immunity to this new virus, the entire human population is the upper limit of possible infections. And although exponential growth always seems slow at first, it can lead to unfathomably high numbers in a rather short time. With the numbers of infected people currently doubling in a matter of a few days[7], our medical systems will be potentially overloaded, which will lead to higher numbers of fatalities, due to people not receiving the care they need.
Door mee te doen met deze beweging beperk je de kans op nieuwe besmettingen, en bescherm je niet alleen jezelf, maar help je ook besmetting en verspreiding naar anderen te voorkomen, in het bijzonder naar diegenen die een groter risico lopen op ernstige gevolgen van dit virus.
Als je dit doel een warm hart toedraagt, deel dit dan op sociale media, via email of vertel erover aan vrienden en familie. En alsjeblieft, blijf verdomme gewoon thuis #StayTheFuckHome.
Ik neem mijn verantwoordelijkheid in het gevecht tegen #COVID19. Ik verzoek je dringend om hetzelfde te doen!

By joining the movement and therefore limiting possibilities for new infections, you are not only protecting yourself, but you are also helping contain and limit the spread for everyone else, especially those who are at higher risk to suffer severe consequences from this virus.
If you care about this cause, please share it on social media, via e-mail or just tell your friends and family about it. But please, just #StayTheFuckHome.
I will do my part in the fight against #COVID19. I urge you to do the same!

```

**Comment by jgroenen** [comment]
Thanks, I'll combine this with my version and make another pull request

**Comment by ahunts** [comment]
Thank you for all of your time and support of the cause!

**Comment by whakkee** [comment]
If you want me to have a look at a PR, I'm Dutch, and would love to help. Ping me if needed!

**Comment by aucuparia** [comment]
@whakkee let's both review the PR once it's submitted so we can get this live as soon as possible.

**Comment by flore2003** [comment]
If any on of you don't mind, you can also create a new PR, we want to get the Dutch version up as soon as possible

**Comment by aucuparia** [comment]
@flore2003 will do now, based on the translation by @jgroenen and the one provided in https://github.com/flore2003/staythefuckhome/issues/3#issuecomment-598728557. These translations are already pretty good, so peer review / fine-tuning can come later IMO.

Give me about half an hour.

**Comment by aucuparia** [comment]
@flore2003 done reviewing @jgroenen's PR. I've added some suggestions. Once those are accepted this translation is ready to go live IMO.

**Comment by jgroenen** [comment]
Please feel free to do so

Op za 14 mrt. 2020 00:47 schreef aucuparia <notifications@github.com>:

**Comment by jgroenen** [comment]
I accepted the suggestions and resolved the merge conflict (lang file). Ready to merge.

**Comment by jgroenen** [comment]
Please feel free to join the Dutch civic tech community at https://doemee.codefor.nl (open Slack environment), #corona

**Comment by whakkee** [comment]
I’ll be home at the end of the afternoon (CET), to look at this - tried from my phone, but that doesn’t work very well. If that takes too long, please just go ahead without me, if there’s something that does need to be changed we can always adjust later as well. But from what I saw so far, it’s looking good. (Bringing my parents groceries so they can stay home)

**Comment by flore2003** [comment]
I've gone ahead and merged the Dutch translation(it's already online), thank you all so much! @whakkee If you have any suggestions, please create a new PR and assign @jgroenen and @aucuparia as reviewers for it. Thank you!

**Comment by aucuparia** [comment]
@flore2003 Great! It seems to be missing some of the suggestions though (which include typos). I'll take another look and submit a PR.
